test.sh

```diff
@@ -1,29 +1,16 @@
-#!/bin/bash
+#!/usr/bin/env bash
 set -xeo pipefail
 
-# A POSIX variable
-OPTIND=1 # Reset in case getopts has been used previously in the shell.
-
-while getopts "d:" opt; do
-    case "$opt" in
-        d)  DOCKER_REPO=$OPTARG
-        ;;
-    esac
-done
-
-if [ "${DOCKER_REPO}" = "" ]; then
-    echo "DOCKER_REPO is required." 1>&2
-    exit 1
-fi
+# Convert Travis-provided repo SLUG to lowercase - Docker's requirement for tags
+SLUG="$(echo "${TRAVIS_REPO_SLUG}" | tr '[:upper:]' '[:lower:]')"
```

Comment:
TRAVIS_REPO_SLUG does not always container repository's URL. Please keep `DOCKER_REPO` in this file.